test/ts/preact.tsx

```diff
@@ -375,13 +379,17 @@ const onBeforeInput = (e: h.JSX.TargetedInputEvent<HTMLInputElement>) => {};
 createElement('input', { onBeforeInput: onBeforeInput });
 h('input', { onBeforeInput: onBeforeInput });
 
+// Should accept onSubmit
 const onSubmit = (e: h.JSX.TargetedSubmitEvent<HTMLFormElement>) => {};
 <form onSubmit={e => e.currentTarget.elements} />;
 createElement('form', { onSubmit: onSubmit });
 h('form', { onSubmit: onSubmit });
 
-h('option', { value: 'foo' });
-createElement('option', { value: 'foo' });
```

Comment:
Sorry for the diff, placed this rather wonkily a few weeks ago. Moved it out so it wasn't sat in the middle of the event tests.